### ISSUE TITLE
Fails to Download "MobInfo2"

### ISSUE THREAD

**Issue by Ironlenny** [ogri-la/strongbox#233]
**Describe the bug**
*A clear and concise description of what the bug is.*

Fails to download "MobInfo2" with the error:
```
failed to download addon, could not install mobinfo2
failed to download file 'https://edge.forgecdn.net/files/3135/377/MobInfo2-8.3.15+Classic.zip': Not Found (HTTP 404)
```

**To Reproduce**
Search for "mobinfo"
Select "Mobinfo2"

**Expected behavior**
The mod should download. I've confirmed that I can download it directly from the Curse site.

**Comment by torkus** [comment]
thanks for opening a ticket, @Ironlenny 

I can replicate this! Always a good start. It's a bit strange to be honest. The URL returned by Curseforge is correct and it downloads using wget ... it might be a url encoding issue with that "`+`" symbol. If so then I'm surprised I haven't come across it earlier.

I'll investigate further.

**Comment by Ironlenny** [comment]
Just confirming that I'm using the Classic WoW client.

**Comment by torkus** [comment]
> Just confirming that I'm using the Classic WoW client.

shouldn't make a difference to strongbox - it just sees a directory of addons.

**Comment by torkus** [comment]
I've taken a look and it may be a bug in a library I'm using. It's certainly unexpected behaviour. I've raised the issue with them and we'll see what they come back with.

**Comment by torkus** [comment]
Fixed in 3.2.1

Thanks for opening a ticket, @Ironlenny
